src/internal/cache-client.ts

```diff
@@ -179,16 +179,12 @@ export class CacheClient {
           interceptors: this.interceptors,
         },
         (err, resp) => {
-          if (resp) {
-            if (resp.missing) {
-              resolve(new SetFetch.Missing());
-            } else if (resp.found) {
-              resolve(new SetFetch.Found(resp.found.elements));
-            } else {
-              resolve(new SetFetch.Error(cacheServiceErrorMapper(err)));
-            }
+          if (resp?.missing) {
+            resolve(new CacheListFetch.Miss());
```

Comment:
Sorry, I didn't get my suggestion fixed in time.

SetFetch.Miss() and you'll have to change SetFetch.Missing to SetFetch.Miss